src/containers/EventListeners/EventListeners.js

```diff
@@ -114,7 +114,7 @@ export /* istanbul ignore next */ class EventListeners extends Component {
         key: 'date',
         header: intl.formatMessage({
           id: 'dashboard.tableHeader.date',
```

Comment:
probably a good idea to update the id too to avoid confusion: `dashboard.tableHeader.dateCreated`
same for the others

Reply:
Done 😸